<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListVehicle</class>
 <widget class="QMainWindow" name="ListVehicle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>790</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/calender.png</normaloff>images/calender.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 5px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}
QTableWidget {
    selection-background-color: #ffcc99;
	selection-color: black;
}
QTableWidget QTableCornerButton::section {
    background: pink;
    border: 2px outset pink;
}
QTableWidget::indicator:unchecked {
    background-color: #d7d6d5
}

QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}
QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lnSearch">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/search.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string> Xóa</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/delete.png</normaloff>images/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string> Gia hạn</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/update.png</normaloff>images/update.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cbbSearch">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Mã thẻ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Biển số</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/refresh.png</normaloff>images/refresh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableMonth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1281</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>Mã thẻ</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Biển số</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Chủ xe</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ngày đăng ký</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ngày hết hạn</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Loại xe</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Giá vé</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trạng thái</string>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string> Đăng ký</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/write.png</normaloff>images/write.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChange">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string> Đổi thẻ</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ticket.png</normaloff>images/ticket.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImport">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string> Đăng ký nhiều</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/excel.png</normaloff>images/excel.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectedFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>35</y>
      <width>700</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>710</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
